ci-operator/step-registry/baremetal/lab/upi/install/baremetal-lab-upi-install-commands.sh

```diff
@@ -241,7 +241,21 @@ compute:
 - architecture: ${architecture}
   hyperthreading: Enabled
   name: worker
-  replicas: ${workers}"
+  replicas: ${workers}
+networking:
```

Comment:
This should go on a patch file of a dedicated step.

The dedicated step can be `baremetal/lab/conf/network/dual-ipv6`.

The step should just print the `networking` stanza into a file that ends with `<_patch_install_config.yaml`. Example: https://github.com/openshift/release/blob/master/ci-operator/step-registry/baremetal/lab/ipi/conf/external-lb/baremetal-lab-ipi-conf-external-lb-commands.sh

You also need to put this snippet in this file, maybe. https://github.com/openshift/release/blob/master/ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh#L152-L159

Reply:
Finally, you should create a workflow for dual stack ipv6 that includes the newly created step (and you could set IPV6_ENABLE to true at that level instead of the test config)

Reply:
Yes. it was only for testing but it should eventually lan on a dedicated workflow

Reply:
Besides DNS & DHCP, it seems that we need also to modify baremetal-lab-pre-console-kargs-commands.sh and kargs related to IP

Not sure how I am saying that based on [Flexy](https://gitlab.cee.redhat.com/aosqe/flexy-templates/-/blob/master/functionality-testing/aos-4_15/hosts/upi_on_baremetal-packet-install.sh?ref_type=heads#L136) comparison

Reply:
I don't think actually, I suggested since I am not familiar with dual stack uPI.
On Flexy implementation above, bonding is used, so what you suggest make sense 

1. Do I need 2 interfaces at least to create a bond
2.  At which stage should it be created. on [Flexy](https://gitlab.cee.redhat.com/aosqe/flexy-templates/-/blob/master/functionality-testing/aos-4_15/hosts/upi_on_baremetal-packet-install.sh?ref_type=heads#L136) example, it passed as kernel args

Reply:
This means also a bond interface should exist on Aux server side, right?

Reply:
Well, I wrongly got the impression that bond Is needed. Given that it's not, this should simplify things.